doc/samples.md

```diff
@@ -164,18 +164,16 @@ steps:
 - script: az extension add -n azure-devops
   displayName: 'Install Azure DevOps Extension'
 
-- script: echo ${AZURE_DEVOPS_CLI_PAT} | az devops login
-  env:
-    AZURE_DEVOPS_CLI_PAT: $(System.AccessToken)
-  displayName: 'Login Azure DevOps Extension'
 
-- script: az devops configure --defaults organization=$(System.TeamFoundationCollectionUri) project=$(System.TeamProject) --use-git-aliases yes
+- script: az devops configure --defaults organization=$(System.TeamFoundationCollectionUri) project=$(System.TeamProject) --use-git-aliases
```

Comment:
Couple of comments: 
1. Configuring project and organization is redundant since the script will run from inside a checked out git repo so project and organization will be picked from there. 
1. Should we mention the use of $(PAT) OR fix permissions of the build account if (System.AccessToken) does not have permission to do something.

Reply:
1. But there is no checking out here in script right? Or do we need to add that as well to provide complete context? 
2. Considering it is an issue (it is an issue right?), we can include that documentation in the troubleshooting guide? Does that make sense? I mean I felt the theme/main value prop here is it help people understand how to use it from YAML.

Reply:
For 1. Git checkout happens in pipeline no matter what.. 
![image](https://user-images.githubusercontent.com/11888714/60098767-e0334380-9773-11e9-8da9-8f2c9e8fa1f0.png)

See list of steps where my yaml contains only a powershell task. Rest of the Tasks are implicit.

Reply:
Capturing notes from our discussion:
1. Will knock off configuration
2. Will include a section in using PAT document to indicate how PATs can be used in pipelines

Reply:
Fixed 1). Will be fixing 2 in official docs